src/connectors/withHistoryActions.jsx

```diff
@@ -1,9 +1,9 @@
+/* eslint-disable require-jsdoc */
```

Comment:
This statements causes eslint error, that says it should be enabled back in the end of the file. Please fix it.

Reply:
Please also fix other eslint errors here.

Reply:
Fixed es lint errors in changed files